### ISSUE TITLE
[4.0 beta1] redeclaration of action_scheduler_register

### ISSUE THREAD

**Issue by yukikatayama** [woocommerce/woocommerce#25676]
**Describe the bug**

Continuing the onboarding steps from #25675, I went and skipped the step to install WC Services. On the last step, when I tried to select a theme or continue with the Twenty Twenty theme, I got a 500 error (I didn't screenshot this however, sorry!). 

Refreshing, gave me fatal: 

```
Fatal error: Cannot redeclare action_scheduler_register_3_dot_0_dot_1() (previously declared in /var/www/wp-content/plugins/mailchimp-for-woocommerce/includes/vendor/action-scheduler/action-scheduler.php:38) in /var/www/wp-content/plugins/woocommerce/packages/action-scheduler/action-scheduler.php on line 37
```

**Comment by juliaamosova** [comment]
Hi @yukikatayama,

Thank you for taking the time to report this bug, we really appreciate your help. The fix is on the way.

**Comment by peterfabian** [comment]
It seems like there was some glitch/incompatibility between MailChimp plugin and WC.
However, MailChimp was updated 4 days ago with the new AS 3.1.0.

I've tried to replicate with MailChimp Version 2.3.2 (which includes AS 3.0.1), but even then, it's a bit strange that the redeclaration would be possible, as the declaration is wrapped in 
```
if ( ! function_exists( 'action_scheduler_register_3_dot_0_dot_1' ) ) { ... }
```

so it shouldn't be possible. Additionally, the function is not declared on line 38 in file `mailchimp-for-woocommerce/includes/vendor/action-scheduler/action-scheduler.php`. I guess it's can't reproduce for now...
